game/templates/game/board.html

```
@@ -302,7 +302,7 @@
             <div class="card">
                 <div class="card-title">Game Controls</div>
                 <div style="display: flex; flex-direction: column; gap: 8px;">
-                    <button class="btn btn-gold" id="flipBtn">Flip Board</button>
+                    <button class="btn btn-gold" id="flipBtn" title="Shortcut: F">Flip Board <span class="btn-shortcut">F</span></button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)